### ISSUE TITLE
ActivityItem A11y Pass

### ISSUE THREAD

**Issue by TestAccessibilityGIT** [OfficeDev/office-ui-fabric-react#7838]
Tracking all of the issues brought up with the 1/28 accessibility pass on ActivityItem control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bed372507-302b-45e2-8c85-8bfcfd546a48%7D&action=default&uid=%7BED372507-302B-45E2-8C85-8BFCFD546A48%7D&ListItemId=1716&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 
  
 - [x] Bug1: [Accessibility][Visual requirements]: Elements must have sufficient color contrast. 
 
 
 Component: ActivityItem 
@micahgodbolt

**Comment by natalieethell** [comment]
It looks like the green color (#00a300) in PersonaInitialsColor is too light to go against the neutralPrimary (#ffffff) background.

https://github.com/OfficeDev/office-ui-fabric-react/blob/c76f02eed90b0c39812451d5a7f947f3bb902628/packages/office-ui-fabric-react/src/components/Persona/PersonaInitialsColor.ts#L59-L60

I'll speak to @betrue-final-final about a potential color change.

**Comment by natalieethell** [comment]
Currently working with designers on color changes for the PersonaCoin.

**Comment by natalieethell** [comment]
We should have the color changes soon!

**Comment by natalieethell** [comment]
This colors are now updated in the fabric-7 branch thanks to @Vitalius1's work in this PR: https://github.com/OfficeDev/office-ui-fabric-react/pull/8781!
